### DISCUSSION TITLE
The copr admin SOP

### DISCUSSION THREAD

**Discussion by pkking** [fedora-copr/copr#2450] [General]
Hi team, is there any copr SOP(such as how to add admin user), as a newbie, want to know about some regular operations as much as possible :)

**Comment by xsuchy** [comment]
https://docs.pagure.org/copr.copr/maintenance_documentation.html

How to add admin is not well documented. It is handled by manage.py - with alias to copr-frontend: 

```
# sudo -u copr-fe copr-frontend --help
```

**Reply by pkking** [reply]
Thanks @xsuchy , i will have a try :)
